docs/pages/model_hub/speaker_diarization.md

```diff
@@ -0,0 +1,34 @@
+# Speaker Diarization (SD) Models
```

Comment:
Do you think it make sense to add Speaker Diarization Models to ASR page of model hub? For example, Hugging Face Hub does exactly that: https://huggingface.co/models?pipeline_tag=automatic-speech-recognition.

Reply:
Speaker diarization can be thought of as a sub-task in ASR or Speaker identification pipelines. However, it is still an independent task and ideally HF should have tags such as "speaker Diarization" and "Speaker Recognition" in their model search.
Once we have next update that adds diarization to transcription, we can add the required info in model hub ASR page. I do not think it makes sense to do it in this PR.

Reply:
Should we rename it to speaker recognition then? So we can put all speaker recognition related models there (speaker diarization, speaker ID etc).

Reply:
Renamed to `speaker_recognition.md`